pyanaconda/payload/live/payload_liveimg.py

```diff
@@ -156,11 +154,15 @@ def post_install(self):
             If file:// was used, just unmount it.
         """
         super().post_install()
-        payload_utils.unmount(IMAGE_DIR)
-        payload_utils.unmount(INSTALL_TREE)
+        task = TearDownMountTask(IMAGE_DIR)
+        task.run()
 
-        if os.path.exists(self.image_path) and not self.data.liveimg.url.startswith("file://"):
```

Comment:
Please explain why the second part of the condition is not needed? AFAICT it was added in response to [rhbz1164585](https://bugzilla.redhat.com/show_bug.cgi?id=1164585) in 290d46bd4c4afae3174089cdc45ac69f4930da4b.

Reply:
The task checks the existence of the file.

Reply:
I am asking why you want to delete the file if it was a local one. Perhaps the part I am not seeing is why the file can be deleted. Do we make a copy of it even if it's a `file:///` kind of url?

Reply:
This condition is still there. Look two lines below.